src/QsCompiler/LanguageServer/EditorState.cs

```diff
@@ -105,6 +105,7 @@ internal bool QsProjectLoader(Uri projectFile, out ProjectInformation info)
             var targetFile = projectInstance.GetPropertyValue("TargetFileName");
             var outputPath = Path.Combine(projectInstance.Directory, outputDir, targetFile);
 
+            var executionTarget = projectInstance.GetProperty("ResolvedQsharpExecutionTarget")?.EvaluatedValue;
```

Comment:
Purely for the sake of using everywhere the same thing: Is there a reason not to use GetPropertyValue?

Reply:
`GetPropertyValue` returns:

> The value of the property with the given name, or an empty string if no property exists with that name.

I used `GetProperty` so that `null` would be the default value instead.

Reply:
Since I decided to make `ScopeContext.ExecutionTarget` non-nullable, I changed this to `GetPropertyValue`.